tasks.yaml

```diff
@@ -54,6 +54,13 @@ tasks:
         setVariables:
           - name: BUILD_ARGS
 
+  # TODO: This is a naive (AND BROKEN!!!) solution. This will only work for a single architecture.
+  #       We need to create subcommands to install specific arch + OS versions of Tofu
+  - name: copy-tofu-bin
+    description: download tofu for go:embed
+    actions:
+      - cmd: cp -Lr $(which tofu) src/cmd/bin/
```

Comment:
should we do a `mkdir -p src/cmd/bin` first here?

Reply:
Also I think we need to adjust file permissions? I ran this and it did move the `tofu` binary but follow up `uds run build-` failed:

```
     cp: src/cmd/bin/tofu: Permission denied
     ERROR:  Failed to run action: command "cp -Lr $(which tofu) src/cmd/bin/" timed out after 0 seconds
```

The permissions I'm seeing are `555` (I'm using Homebrew for my `tofu` install)

Reply:
Good call on the `mkdir`, I forgot that I manually made that myself.

The permission decided is another good catch. That is happening because the destination file already exists. I'll add the `-f` flag to the cp command.